test/specs/anInitialSuite.e2e.ts

```diff
@@ -96,40 +95,40 @@ describe('An Initial Suite', async () => {
-      await utilities.enableAllExtensions();
-      await utilities.executeQuickPick('Extensions: Show Enabled Extensions', Duration.seconds(2));
-      const prompt = await utilities.openCommandPromptWithCommand(workbench, 'SFDX:');
-      const quickPicks = await prompt.getQuickPicks();
-      const commands = await Promise.all(quickPicks.map((quickPick) => quickPick.getLabel()));
-
-      // Look for the first few SFDX commands.
-      await expect(commands).toContain('SFDX: Create Project');
-      await expect(commands).toContain('SFDX: Authorize a Dev Hub');
-      await expect(commands).toContain('SFDX: Authorize an Org');
-      await expect(commands).toContain('SFDX: Authorize an Org using Session ID');
-      await expect(commands).toContain('SFDX: Cancel Active Command');
-      await expect(commands).toContain('SFDX: Configure Apex Debug Exceptions');
-      await expect(commands).toContain('SFDX: Create a Default Scratch Org...');
-      await expect(commands).toContain('SFDX: Create and Set Up Project for ISV Debugging');
-      await expect(commands).toContain('SFDX: Create Apex Class');
-      await expect(commands).toContain('SFDX: Create Apex Trigger');
-      // There are more, but just look for the first few commands.
-
-      // Escape out of the pick list.
-      await prompt.cancel();
-    }
-  );
+  step('Verify that SFDX commands are present after an SFDX project has been created', async () => {
+    const workbench = await utilities.getWorkbench();
+    await utilities.enableAllExtensions();
+    await utilities.executeQuickPick(
+      'Extensions: Show Enabled Extensions',
+      utilities.Duration.seconds(2)
+    );
+    const prompt = await utilities.openCommandPromptWithCommand(workbench, 'SFDX:');
+    const quickPicks = await prompt.getQuickPicks();
+    const commands = await Promise.all(quickPicks.map((quickPick) => quickPick.getLabel()));
+
+    // Look for the first few SFDX commands.
+    await expect(commands).toContain('SFDX: Create Project');
+    await expect(commands).toContain('SFDX: Authorize a Dev Hub');
+    await expect(commands).toContain('SFDX: Authorize an Org');
+    await expect(commands).toContain('SFDX: Authorize an Org using Session ID');
+    await expect(commands).toContain('SFDX: Cancel Active Command');
+    await expect(commands).toContain('SFDX: Configure Apex Debug Exceptions');
+    await expect(commands).toContain('SFDX: Create a Default Scratch Org...');
+    await expect(commands).toContain('SFDX: Create and Set Up Project for ISV Debugging');
+    await expect(commands).toContain('SFDX: Create Apex Class');
+    await expect(commands).toContain('SFDX: Create Apex Trigger');
+    // There are more, but just look for the first few commands.
+
+    // Escape out of the pick list.
+    await prompt.cancel();
+  });
 
   after('Tear down and clean up the testing environment', async () => {
-    await testSetup.tearDown();
+    await testSetup?.tearDown();
```

Comment:
Is this safe navigation operator a change we would like to make to all E2E tests or just AnInitialSuite as a special case?

Reply:
It might be worthwhile, but no rush.